standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestMetaStoreEndFunctionListener.java

```diff
@@ -83,7 +86,7 @@ public void testEndFunctionListener() throws Exception {
     listSize = DummyEndFunctionListener.funcNameList.size();
     String func_name = DummyEndFunctionListener.funcNameList.get(listSize-1);
     MetaStoreEndFunctionContext context = DummyEndFunctionListener.contextList.get(listSize-1);
-    assertEquals(func_name,"get_database");
+    assertEquals(func_name,"get_database_req");
```

Comment:
after this change it seems like `EndFunctionListers` will get a bit different data - is that true?

Reply:
Thanks a lot for the feedback! you are right, this is where may break compatibility, including the MetaStoreEndFunctionContext(we cannot retrieve inputTableName by method `getInputTableName()` any more). I not so sure if we can make such changes though this can help clean some codes...